src/Service/EmailNotificationService.php

```diff
@@ -61,14 +61,16 @@ public function sendEmails()
                         'count' => count($participants)
                     ]);
                 } else {
-                    $message = new Message($this->env, $this->loggerService, $this->twig, $this->params);
-                    $message
-                        ->setTo($result['emails'])
-                        ->render($this->render, [
-                            $this->level => $result['id'],
-                            'participants' => $participants
-                        ])
-                        ->send();
+                    if ($this->env->isStable() || $this->env->isProd()) {
```

Comment:
This won't log the message in the logs as the send method is not called. I think the best way to do this is to remove the mail_method key from configurations, which will then prevent sending emails but still log.

Below is the code where the mail method is checked.
https://github.com/all-of-us/healthpro/blob/develop/src/Service/Message.php#L81

Reply:
You're right. Good call I didn't notice that.